node/bft/examples/simple_node.rs

```diff
@@ -129,19 +129,20 @@ pub async fn start_bft(
         Arc::new(BFTMemoryService::new()),
         BatchHeader::<CurrentNetwork>::MAX_GC_ROUNDS as u64,
     );
-    // Initialize the gateway IP and dev mode.
-    let (ip, dev) = match peers.get(&node_id) {
-        Some(ip) => (Some(*ip), None),
-        None => (None, Some(node_id)),
+    // Initialize the gateway IP and storage mode.
+    let ip = match peers.get(&node_id) {
+        Some(ip) => Some(*ip),
+        None => Some(SocketAddr::from_str(&format!("127.0.0.1:{}", MEMORY_POOL_PORT + node_id)).unwrap()),
```

Comment:
Instead of `from_str`, it would be a littler nicer to do the following:

 ```rust
SocketAddr::new(IpAddr::V4(IPv4::LOCALHOST), MEMORY_POOL_PORT + node_id)
```
That also avoids the need for `unwrap()`.

Reply:
Fixed in https://github.com/ProvableHQ/snarkOS/pull/3513/commits/534a29bbcf8d66c9aec23da3ffaa4ce3cc0f0695 along with a few other instances.